salsa_dancing_molecules/cohesive_energy.py

```diff
@@ -0,0 +1,35 @@
+"""Calculate cohesive energy."""
+
+from asap3 import Trajectory
+
+
+def get_cohesive_energy(traj_file, t0):
+    """Calculate cohesive energy.
+
+    Input:
+        traj_file: str  - filename of trajectory file
+        t0: int         - timestep when equilibrium starts
+
+    Output:
+        cohesive energy for the system (in eV/atom): float
+
+    """
+    configs = Trajectory(traj_file)
+    pot_energies = [atom.get_potential_energy() for atom in configs]
+    avg_pot_e = (sum(pot_energies[t0:]) / len(pot_energies[t0:]))
+
+    return calculate_cohesive_energy(configs, avg_pot_e)
+
+
+def calculate_cohesive_energy(configs, avg_pot_e):
+    """Calculate cohsive energy from potential energy.
```

Comment:
Is it true that the average potential energy per atom is the same as the cohesive energy? I think this calculation is only true for atoms that have reached equilibrium. Maybe change to this comment to "Calulate cohesive energy from potential energy in equilibrium" or change the description of avg_pot_e to "-average potential energy at equilibrium"